plugins/jamf/plugin.spec.yaml

```diff
@@ -186,20 +187,23 @@ types:
 connection:
   url:
     title: Jamf URL
-    description: The full URL for your Jamf instance, e.g. https://instance.jamfcloud.com
+    description: The full URL for your Jamf instance, e.g. https://{instance}.jamfcloud.com
```

Comment:
Is example still needed here if its included in the example field below?

Reply:
I just left that as it is, like for customer to make sure what to put inside the field, but it's fine for me to remove that from description. What do you think? 😄

Reply:
I think if its in example then we don't need it twice

Reply:
Updated 😄